app/mailers/school_mailer.rb

```diff
@@ -278,4 +279,18 @@ def year2020_invite_email(recipient:, start_url:)
       },
     )
   end
+
+  def year2020_add_participants_confirmation(school:, participants:)
+    @school = school
+    @participants = participants
+
+    view_mail(
+      BASIC_TEMPLATE,
+      to: school.contact_email,
+      subject: "2020 to 2021 NQTs cohort: support materials confirmation",
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      template_name: :year2020_ects_added_confirmation,
+    )
+  end
```

Comment:
Why are we doing this this way? This is not how we've done any other email on this service, and defeats all of the security that templating gives you...

Reply:
I believe this is because we needed a dynamic list of trainee names to be rendered, which wasn't possible to do with the templating that notify gives you.